.github/workflows/rust.yml

```diff
@@ -0,0 +1,25 @@
+on: [push, pull_request]
+
+name: build
+
+jobs:
+  check:
+    name: Rust project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install latest nightly
+        uses: actions-rs/toolchain@v1
+        with:
+            toolchain: nightly
+            override: true
+            components: rustfmt, clippy
+
+      - name: Check
+        run: cargo check --verbose --all
+      - name: Build
+        run: cargo build --verbose
+      - name: Clippy
+        run: cargo clippy --verbose --all --all-targets --all-features -- -D warnings -D clippy::clone_on_ref_ptr -D clippy::enum_glob_use
+      - name: Fmt
+        run: cargo fmt --verbose --all -- --check
```

Comment:
Miss `cargo test` here.
You can use `docker-compose` in github actions to run a muta chain for testing.

ref:
- https://github.com/peter-evans/docker-compose-actions-workflow
- https://github.com/nervosnetwork/muta/blob/e84756d1734ad58943309c3c2299393f5a2022e4/devtools/docker/dockerfiles/Dockerfile.dev